src/client/pages/OfferNew/Introduction/Sidebar/StartDate.tsx

```diff
@@ -342,8 +347,10 @@ export const StartDate: React.FC<Props> = ({
 }) => {
   const textKeys = useTextKeys()
   const [showError, setShowError] = React.useState(false)
+
+  // TODO: Make this flag more generic. This logic should not live here.
   const isSingleStartDateBundle = (offerData: OfferData) =>
-    isBundle(offerData) && isDanish(offerData)
+    isBundle(offerData) && (isDanish(offerData) || isSwedish(offerData))
```

Comment:
@gustaveen the code that decided whether we show one or two date selectors is here ☝️ 
I guess it’s okey to merge the PR and then make changes if we need to show 2 date selectors later in SE.

Reply:
Sounds great! 🙌